src/navigation/navigation.es6

```diff
@@ -159,7 +159,9 @@ const initLoginButton = (root) => {
 
       state.currency = data.authorize.currency;
       local_storage.set('currency', state.currency);
-      update_balance(data);
+      const loginId = local_storage.get('authorize').loginid;
+      if (data.authorize && data.authorize.loginid === loginId) update_balance(data);
```

Comment:
LGTM and I'm going to approve it but I've got 2 nitpicks.
- `loginId` or `login_id`? What's the project convention?
- Could we use `data.authorize?.loginid`?

Reply:
thanks salar for your fast and not furious approval :)
we do not have certain convention for variables in webtrader and since I used camelCase  before, I keep it like before.
also webtrader do not support optional chaining, so we could not use `data.authorize?.loginid?` .

Reply:
Thank you for the explanation :))